Sources/Build/BuildDescription/ClangTargetBuildDescription.swift

```diff
@@ -305,6 +311,16 @@ package final class ClangTargetBuildDescription {
             args += ["-I", includeSearchPath.pathString]
         }
 
+        // FIXME: Remove this once it becomes possible to express this dependency in a package manifest.
+        //
+        // On Linux/Android swift-corelibs-foundation depends on dispatch library which is
+        // currently shipped with the Swift toolchain.
+        if (triple.isLinux() || triple.isAndroid()) && self.package.id == .plain("swift-corelibs-foundation") {
```

Comment:
I don't remember off the top of my head if we have `isDarwin()` on triples, but I'd expect it tool like this:
```suggestion
        if !triple.isDarwin() && self.package.id == .plain("swift-corelibs-foundation") {
```

Reply:
I re-created the check exactly how it’s in the package file which covers only Linux and Android.